src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -76,7 +77,7 @@ public PSRoleDefinition GetRoleDefinition(string roleId)
         {
             return AuthorizationManagementClient.RoleDefinitions.GetById(roleId)
 #if !NETSTANDARD
-                .RoleDefinition
+                //.RoleDefinition
```

Comment:
Because in the new 2.5 authorization package there has no function 'RoleDefinitionGetResult'  and we can't use '.RoleDefinition' in project.